### ISSUE TITLE
Running Error

### ISSUE THREAD

**Issue by Daniel-1234567** [flexflow/FlexFlow#2]
Hi, I tried several times to run this project, but I still fail to run it. There are some errors as below:

`sheldon@instance-2:~/FlexFlow$ ./ffcompile.sh alexnet
---> Linking objects into one binary: alexnet
g++ -o alexnet cnn.cc.o cnn_mapper.cc.o alexnet.cc.o conv_2d.cu.o ops.cu.o pool_2d.cu.o batch_norm.cu.o linear.cu.o softmax.cu.o concat.cu.o cnn_helper.cu.o -lcudnn -lcublas -lcurand  -L. -llegion -lrealm -lrt -lpthread -ldl -rdynamic -L/usr/local/cuda/lib64 -L/usr/local/cuda/lib64/stubs -lcuda -Xlinker -rpath=/usr/local/cuda/lib64 -lz 
conv_2d.cu.o: In function `__sti____cudaRegisterAll()':
tmpxft_000029b2_00000000-5_conv_2d.cudafe1.cpp:(.text.startup+0x19): undefined reference to `__cudaRegisterFatBinaryEnd'
ops.cu.o: In function `__sti____cudaRegisterAll()':
tmpxft_000029b8_00000000-5_ops.cudafe1.cpp:(.text.startup+0x9b): undefined reference to `__cudaRegisterFatBinaryEnd'
pool_2d.cu.o: In function `__sti____cudaRegisterAll()':
tmpxft_000029b9_00000000-5_pool_2d.cudafe1.cpp:(.text.startup+0x19): undefined reference to `__cudaRegisterFatBinaryEnd'
batch_norm.cu.o: In function `__sti____cudaRegisterAll()':
tmpxft_000029b6_00000000-5_batch_norm.cudafe1.cpp:(.text.startup+0x19): undefined reference to `__cudaRegisterFatBinaryEnd'
linear.cu.o: In function `__sti____cudaRegisterAll()':
tmpxft_000029b7_00000000-5_linear.cudafe1.cpp:(.text.startup+0x19): undefined reference to `__cudaRegisterFatBinaryEnd'
softmax.cu.o:tmpxft_000029e9_00000000-5_softmax.cudafe1.cpp:(.text.startup+0x47): more undefined references to `__cudaRegisterFatBinaryEnd' follow
./librealm.a(cuda_module.cc.o): In function `Realm::Cuda::GPU::create_dma_channels(Realm::RuntimeImpl*)':
cuda_module.cc:(.text+0x20e7): undefined reference to `Realm::register_gpu_in_dma_systems(Realm::Cuda::GPU*)'
collect2: error: ld returned 1 exit status
/home/sheldon/install/legion/runtime/runtime.mk:601: recipe for target 'alexnet' failed
make: *** [alexnet] Error 1
sheldon@instance-2:~/FlexFlow$ 
`

Could you please help me with this? Thank you very much.
